jdat_notebooks/background_estimation_imaging/Imaging Sky Background Estimation.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<p>this tries to download the movie on my machine instead of showing a video inline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
It depends on your browser. We can stick with this for now. I need to explore other video display commands.

Reply:
Ok, I agree this is fine for now then.  We would probably need to host on a hosting site (youtube etc) to get around browser-dependent behavior, but I don't think downloading a video and playing manually is a dealbreaker.